conversation_reconstruction/dataflow/conversation_reconstruction/dataflow_clean_output.py

```diff
@@ -15,79 +15,77 @@
 
 -------------------------------------------------------------------------------
 
-A dataflow pipeline to collect statistics on Wikipedia talk pages including number of revisions and their sizes.
+A dataflow pipeline to clean scored outputs.
```

Comment:
Say a little about what that means. Removing wiki markup and tokenization?

Reply:
I think this to clean the outputs scored using perspective api internally, might be better to not include this code.